core/src/main/java/com/graphhopper/routing/profiles/MappedDecimalEncodedValue.java

```diff
@@ -0,0 +1,53 @@
+package com.graphhopper.routing.profiles;
+
+import com.carrotsearch.hppc.IntIntHashMap;
+
+import java.util.List;
+
+public class MappedDecimalEncodedValue extends IntEncodedValue {
+    private final int toValueMap[];
+    private final IntIntHashMap toStorageMap;
+    private final double precision;
+
+    /**
+     * TODO should we really use precision here or use something like the already used 'factor'?
```

Comment:
Mhm, precision is the commonly used word for Double, so this does not sound wrong to me.